src/component/Calculator.js

```diff
@@ -0,0 +1,48 @@
+import React from 'react';
+
+function App() {
```

Comment:
- [ ] This component name should be Calculator as specified in the requirements. We already have an App component, there is no need creating another one.

- [ ] Kindly note that this Calculator component should have at least one child component. Kindly create that.